pkg/apis/core/types.go

```diff
@@ -2160,6 +2160,9 @@ type ResourceRequirements struct {
 	// otherwise to an implementation-defined value
 	// +optional
 	Requests ResourceList
+	// The entries are the names of resources in PodSpec.ResourceClaims
+	// that are used by the container.
```

Comment:
Add a note that this is an alpha-field and which kube_feature controls it. Same for the other new fields.

Reply:
Although the [corresponding KEP](https://github.com/kubernetes/enhancements/issues/2558) hasn't made progress? I had checked and it looked like the effort to add that syntax had been abandoned.

Just wondering. I have added it to both internal and v1 types.go, just to be on the safe side. It probably only really needs to be in the external one?